File: BotsSDK/korebot/src/main/java/kore/botssdk/audiocodes/webrtcclient/Activities/SettingsActivity.java
package kore.botssdk.audiocodes.webrtcclient.Activities;

import android.os.Bundle;
import android.view.View;

import kore.botssdk.R;

public class SettingsActivity extends BaseAppCompatActivity {


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        //this.requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.settings_activity);

        initGui();
    }

    private void initGui() {

        int[] settingsButtonClickListID = {R.id.settings_button_general, R.id.settings_button_account, R.id.settings_button_call_stats};

        View.OnClickListener settingsClickListener = new View.OnClickListener() {

            @Override
            public void onClick(View clickedView) {
                if (clickedView == null) {
                    return;
                }
                int id = clickedView.getId();
                if (id == R.id.settings_button_general) {
                    startNextActivity(GeneralSettingsActivity.class);
                } else if (id == R.id.settings_button_account) {
                    startNextActivity(AccountActivity.class);
                } else if (id == R.id.settings_button_call_stats) {
                    startNextActivity(CallStatsActivity.class);
                }

            }
        };

        for (int settingsButtonClickID : settingsButtonClickListID) {
            View view = findViewById(settingsButtonClickID);
            if (view != null) {
                view.setOnClickListener(settingsClickListener);
            }
        }
    }
//
//
//    private void openNextScreen(Class cls) {
//        //start login and open app main screen
//        //ACManager.getInstance().startLogin();
//        startNextActivity(cls);
//    }

}

File: BotSdk/ui/src/main/java/com/kore/ui/audiocodes/webrtcclient/activities/SettingsActivity.kt
package com.kore.ui.audiocodes.webrtcclient.activities

import android.os.Bundle
import android.view.View
import androidx.databinding.DataBindingUtil
import com.kore.ui.R
import com.kore.ui.audiocodes.webrtcclient.activities.AccountActivity
import com.kore.ui.audiocodes.webrtcclient.activities.CallStatsActivity
import com.kore.ui.audiocodes.webrtcclient.activities.GeneralSettingsActivity
import com.kore.ui.databinding.SettingsActivityBinding

class SettingsActivity : BaseAppCompatActivity() {
    private lateinit var binding: SettingsActivityBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = DataBindingUtil.inflate(layoutInflater, R.layout.settings_activity, null, false)
        setContentView(binding.root)
        initGui()
    }

    private fun initGui() {
        binding.settingsButtonGeneral.setOnClickListener { startNextActivity(GeneralSettingsActivity::class.java) }
        binding.settingsButtonAccount.setOnClickListener { startNextActivity(AccountActivity::class.java) }
        binding.settingsButtonCallStats.setOnClickListener { startNextActivity(CallStatsActivity::class.java) }
    }
}